.github/workflows/dist.yml

```diff
@@ -4,17 +4,25 @@ on:
   push:
     tags:
       - "v[0-9]+.[0-9]+.[0-9]+"
+  workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    container:
+      image: registry.fedoraproject.org/fedora:latest
+      env:
+        PATH: /root/.local/bin:/root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    defaults:
+      run:
+        shell: bash
```

Comment:
what are these settings needed? we use container actions in workflows of other projects, and I don't see anything that doesn't work or broke because of the lack of those